build/postgres-ha/Dockerfile

```diff
@@ -117,38 +75,33 @@ RUN useradd crunchyadm -g 0 -u 17
 
 ENV PATH="${PGROOT}/bin:${PATH}"
 
-RUN mkdir -p /opt/cpm/bin /opt/cpm/conf /pgdata /pgwal /pgconf /backrestrepo \
+RUN mkdir -p /opt/crunchy/bin /opt/crunchy/conf /pgdata /pgwal /pgconf /backrestrepo \
 		/crunchyadm /tablespaces
 
 # Adjust ownership for the folders to be the "postgres" user and allow the group
 # permissions to match the user ones EXCEPT for the /tablespaces folder, which
 # will only have permissions on the user
-RUN chown -R postgres:postgres /opt/cpm /var/lib/pgsql \
+RUN chown -R postgres:postgres /opt/crunchy /var/lib/pgsql \
 	/pgdata /pgwal /pgconf /backrestrepo /crunchyadm /tablespaces &&  \
-	chmod -R g=u /opt/cpm /var/lib/pgsql \
+	chmod -R g=u /opt/crunchy /var/lib/pgsql \
 	/pgdata /pgwal /pgconf /backrestrepo /crunchyadm /tablespaces
 
 # open up the postgres port
 EXPOSE 5432
 
-ADD bin/postgres-ha /opt/cpm/bin
-ADD bin/common /opt/cpm/bin/common
-ADD conf/postgres-ha /opt/cpm/conf
-ADD tools/pgmonitor/exporter/postgres /opt/cpm/bin/modules/pgexporter
-
-ADD yq /opt/cpm/bin
-RUN chmod +x /opt/cpm/bin/yq
-
-RUN chmod g=u /etc/passwd \
- && chmod g=u /etc/group
+ADD bin/postgres-ha /opt/crunchy/bin/postgres-ha
+ADD bin/common /opt/crunchy/bin/common
+ADD conf/postgres-ha /opt/crunchy/conf/postgres-ha
+ADD tools/pgmonitor/exporter/postgres /opt/crunchy/bin/modules/pgexporter
```

Comment:
This may be able to be added as part of the `postgres` image. currently the two images `postgres` and `postgres-ha` put the modules in different places (`/opt/crunchy/bin/postgres/modules` and `/opt/crunchy/bin/modules`). I think updating them to both use `/opt/crunchy/bin/modules` is a worthwhile change